Dart/src/com/jetbrains/lang/dart/ide/completion/DartServerCompletionContributor.java

```diff
@@ -228,9 +228,9 @@ private static void addToCompletionList(@NotNull Project project,
       if (!addedCompletions.add(suggestion)) continue;
 
       Element element = suggestion.getElement();
-      String libraryUri = element != null ? element.getLibraryUri() : null;
+      String libraryUri = suggestion.getLibraryUri();
       String libraryUriToImport =
-        element != null && suggestion.getIsNotImported() != null && suggestion.getIsNotImported() ? element.getLibraryUri() : null;
+        element != null && suggestion.getIsNotImported() != null && suggestion.getIsNotImported() ? libraryUri : null;
```

Comment:
Would `suggestion.getIsNotImported() == true` or `suggestion.getIsNotImported() == Boolean.TRUE` work here?

Reply:
`suggestion.getIsNotImported() == true` will throw NPE, `suggestion.getIsNotImported() == Boolean.TRUE` is fine. I ended up with
`String libraryUriToImport = suggestion.getIsNotImported() == Boolean.TRUE ? libraryUri : null;`